### DISCUSSION TITLE
Welcome to design Discussions!

### DISCUSSION THREAD

**Discussion by dschuff** [WebAssembly/design#1447] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.
 
These discussions (like all other WebAssembly CG activity) are subject to the CG's [Code of Conduct](https://github.com/WebAssembly/.github/blob/main/CODE_OF_CONDUCT.md).

**Comment by dschuff** [comment]
We've discussed enabling this feature a couple of times now, some of the repos have it enabled, and it seems to work well. For the design repo in particular, I suspect most of the things people have posted as issues are better suited as discussions.

**Comment by syrusakbary** [comment]
Nice! Happy to have this enabled :)
